<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMoneyQifProfileEditor</class>
 <widget class="QWidget" name="MyMoneyQifProfileEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QIF Profile Editor</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QListWidget" name="m_profileListBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="profileTabs">
       <widget class="QWidget" name="tabGeneral">
        <attribute name="title">
         <string comment="@title QIF profile general information">General</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QGridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="TextLabel2_2_3">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Void mark</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="TextLabel2_2">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Opening Balance text</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="m_editOpeningBalance"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="TextLabel2">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Type field text</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="m_editAccountDelimiter"/>
           </item>
           <item row="4" column="1">
            <widget class="KLineEdit" name="m_editVoidMark"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="TextLabel2_2_2">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Account delimiter</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KLineEdit" name="m_editType"/>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="m_editDescription"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="TextLabel1">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="m_attemptMatch">
           <property name="text">
            <string>Attempt to match similar transactions</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="Spacer12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabFilter">
        <attribute name="title">
         <string>Filter</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QGridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="TextLabel1_4_2">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Output filter location</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KUrlRequester" name="m_editInputFilterLocation"/>
           </item>
           <item row="2" column="1">
            <widget class="KUrlRequester" name="m_editOutputFilterLocation">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="TextLabel1_4_3">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Input filter file type</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KLineEdit" name="m_editInputFilterFileType">
             <property name="text">
              <string>*.qif</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="TextLabel1_4">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Input filter location</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="Spacer9_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>90</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabDate">
        <attribute name="title">
         <string>Date</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QHBoxLayout">
           <item>
            <layout class="QGridLayout">
             <item row="0" column="1">
              <widget class="KComboBox" name="m_editDateFormat">
               <property name="whatsThis">
                <string>The format of the dates in the QIF file.</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="TextLabel3">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>0</height>
                </size>
               </property>
               <property name="whatsThis">
                <string>The format of the dates in the QIF file.</string>
               </property>
               <property name="text">
                <string>Date Format</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="KComboBox" name="m_editApostrophe">
               <property name="whatsThis">
                <string>If a QIF file contains date entries with years represented with two digits then either an apostrophe or a slash may be used to delimit the year in the dates for certain centuries.  This enables 1905 to be distinguished from 2005.  Specify here which range of years will have year delimited by an apostrophe (eg. for Quicken this is usually 1900-1949).</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="TextLabel4">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>0</height>
                </size>
               </property>
               <property name="whatsThis">
                <string>If a QIF file contains date entries with years represented with two digits then either an apostrophe or a slash may be used to delimit the year in the dates for certain centuries.  This enables 1905 to be distinguished from 2005.  Specify here which range of years will have year delimited by an apostrophe (eg. for Quicken this is usually 1900-1949).</string>
               </property>
               <property name="text">
                <string>Apostrophe Handling</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="Spacer9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>90</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="Spacer8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>80</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabMoney">
        <attribute name="title">
         <string>Amounts</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QTreeWidget" name="m_editAmounts">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string comment="@title header of the QIF field name column">Field</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>QIF-Record</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Decimal Symbol</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Thousand Delimiter</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>SortColumnInvisible</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Value</string>
            </property>
            <property name="text">
             <string>T</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Split-Value</string>
            </property>
            <property name="text">
             <string>$</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Commission</string>
            </property>
            <property name="text">
             <string>O</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Price</string>
            </property>
            <property name="text">
             <string>I</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Quantity</string>
            </property>
            <property name="text">
             <string>Q</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <layout class="QGridLayout">
             <item row="1" column="1">
              <widget class="KComboBox" name="m_thousandsBox"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="TextLabel1_2">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Decimal Symbol</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="TextLabel1_3">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Thousands Separator</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="KComboBox" name="m_decimalBox"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="Spacer7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="Spacer8_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_newButton">
       <property name="text">
        <string comment="@action create new profile">New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_renameButton">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>104</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_profileListBox</tabstop>
  <tabstop>profileTabs</tabstop>
  <tabstop>m_editDescription</tabstop>
  <tabstop>m_editType</tabstop>
  <tabstop>m_editOpeningBalance</tabstop>
  <tabstop>m_editAccountDelimiter</tabstop>
  <tabstop>m_editVoidMark</tabstop>
  <tabstop>m_helpButton</tabstop>
  <tabstop>m_newButton</tabstop>
  <tabstop>m_deleteButton</tabstop>
  <tabstop>m_renameButton</tabstop>
  <tabstop>m_resetButton</tabstop>
  <tabstop>m_editDateFormat</tabstop>
  <tabstop>m_editApostrophe</tabstop>
  <tabstop>m_decimalBox</tabstop>
  <tabstop>m_thousandsBox</tabstop>
  <tabstop>m_editAmounts</tabstop>
  <tabstop>m_editInputFilterLocation</tabstop>
  <tabstop>m_editOutputFilterLocation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
